spacy/errors.py

```diff
@@ -914,7 +914,7 @@ class Errors(metaclass=ErrorsWithCodes):
     E1034 = ("Node index {i} out of bounds ({length})")
     E1035 = ("Token index {i} out of bounds ({length})")
     E1036 = ("Cannot index into NoneNode")
-
+    E1037 = ("Invalid attribute value '{attr}'.")
```

Comment:
Sorry, missed this the first time:

```suggestion
    E1037 = ("Invalid attribute value '{attr}'.")

```